### PR TITLE
ES Homework

### PR DESCRIPTION
**Pull request by EvanSanderson**
Comfort Level: 4
Completeness: 5
I ended up really liking this assignment - its great to get the practice. Were we supposed to have done the parts utilizing Sinatra as well?

### CODE REVIEW

**Comment by EvanSanderson** [comment]
completeness: 5
comfort level: 5

feelin good and ready for rails!

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  tenant = Tenant.find(params[:id])
+  tenant.update(params[:tenant])
+  redirect "/tenant/#{tenant.id}"
+end
+
+delete '/tenant/:id' do
+  tenant = Tenant.find(params[:id])
+  tenant.destroy
+  redirect '/tenant'
+end
```
Comment (nayana487) [inline app.rb:91]: nice job setting up your routes! This gets tricky with the dynamically changing ids, but you'll see that this is much easier with rails.

File: ar_exercise/exercise.rb [hunk 2]
```diff
@@ -34,38 +34,77 @@ class Apartment < ActiveRecord::Base
...
 
 # Birthday!
 # It's Kristin Wisoky's birthday. Find her in the DB and change her age to be 1 year older
 # Note: She's in the seed data, so she should be in your DB
-
+kristin = Tenant.find_by(name: "Kristin Wisoky");
+kristin.update(age: 24)
 # Rennovation!
 # Find the apartment "62897 Verna Walk" and update it to have an additional bedroom
+verna = Apartment.find_by(address: "62897 Verna Walk");
```
Comment (nayana487) [inline ar_exercise/exercise.rb:92]: nice, you definitely have a good handle on AR queries.

File: views/apartment/index.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h2>| Apartment's You Are Currently Managing | </h2>
+<a href="/apartment/new"> Make a New Apartment </a>
+<% @apartments.each do |apartment| %>
+<div class="apartments">
+  <p><%=apartment.address%></p>
+  <p>$<%=apartment.monthly_rent%></p>
+  <a href="/apartment/<%=apartment.id%>"><p> Click here to learn more information about the apartment.</p></a>
```
Comment (nayana487) [inline views/apartment/index.erb:7]: again, be mindful of spacing with erb tags and the html ones in this case.

File: views/apartment/show.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+<p> The number of beds is: </p>
+<p><%=@apartment.num_beds%></p>
+<p>***************************<p>
+<p> The number of baths is </p>
+<p><%=@apartment.num_baths%></p>
+<p>***************************<p>
+<p> The tenants are: </p>
+<%@apartment.tenants.each do |tenant|%>
+  <p> <%=tenant.name%> </p>
+  <%end%>
```
Comment (nayana487) [inline views/apartment/show.erb:20]: nice job implementing your associations here.

**Comment by nayana487** [comment]
great work! you definitely got the landlord functionality down.

**Pull request closed without merging**